### DISCUSSION TITLE
Is "Failed" a required field in SendMessageBatch response

### DISCUSSION THREAD

**Discussion by Dmdl** [boto/botocore#2937] [Q&A] [answered]
I'm trying to mock `send_message_batch` with the below response, but `botocore` is unhappy that I'm not including the "Failed" field and throws a ParamValidationError.

```
{
          "Successful": [
              {
                  "Id": "string",
                  "MD5OfMessageAttributes": "string",
                  "MD5OfMessageBody": "string",
                  "MD5OfMessageSystemAttributes": "string",
                  "MessageId": "string",
                  "SequenceNumber": "string"
              }
          ],
      }
```
```
botocore.exceptions.ParamValidationError: Parameter validation failed:
Missing required parameter in input: "Failed"
```

I'm I doing anything wrong?

**Comment by Dmdl** [comment]
Found an old ticket confirming that this is a bug - https://github.com/boto/botocore/issues/1782

**Reply by RyanFitzSimmonsAK** [reply, marked as answer]
Hi @Dmdl, thanks for reaching out.  It looks like the issue you created has already been triaged and a tracking issue created in the cross-SDK repository.  I'm going to close this discussion, and as mentioned, you can refer to the tracking issue for further updates.  Thanks!
